polyply/src/minimizer.py

```diff
@@ -183,7 +183,8 @@ def target_function(positions):
                 energy += new
         return energy
 
-    opt_results = scipy.optimize.minimize(target_function, positions, method='L-BFGS-B',
+    opt_results = scipy.optimize.minimize(target_function, np.ravel(positions),
+                                          method='L-BFGS-B',
```

Comment:
@csbrasnett can you remove the np.ravel from this line? Instead put the following in line 169:
```
positions = np.ravel(np.array(list(coords.values())))
```